package.json

```diff
@@ -44,6 +44,6 @@
     "cz-conventional-changelog": "3.3.0",
     "prettier": "3.3.2",
     "prettier-plugin-tailwindcss": "0.6.5",
-    "typescript": "~5.5.3"
+    "typescript": "~5.5.0"
```

Comment:
downgrade?

Reply:
Yes. Our ESLint plugin does not support 5.5.3 yet :)

Reply:
I can make an npm update and check if an updated version supports it, but... let me check.